data/mods/blazemod/items/vehicle/blob_vehicleparts.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
One of the re-writes here, but should be noted as modernisation- biggest change is the name of the frost_hull (id remains same) to `"gelacier boat hull"` from `"gelacier wheel"`, copying the actual frostie_wheel. Small symbol change to bring to parity with other hull-type items. Surface level changes only, and separated out here due to a lack of other appropriate populated categories.